python/openmldb/dbapi/dbapi.py

```diff
@@ -362,7 +357,24 @@ def fetchone(self):
 
     @connected
     def fetchmany(self, size=None):
-        raise NotSupportedError("Unsupported in OpenMLDB")
+        if self._resultSet is None: raise Exception("query data failed")
+        if size is None:
```

Comment:
`size` is not initialized and will be None all the time. Maybe just init with `size = self.arraysize`?

Reply:
the default args of class function does not support the class attribute. `def fetchmany(self, size = self.arraysize):`

Reply:
`arraysize` means all rows? But I only see `rowcount` updated, where to update arraysize?